src/shader_recompiler/frontend/translate/data_share.cpp

```diff
@@ -65,4 +65,20 @@ void Translator::V_READFIRSTLANE_B32(const GcnInst& inst) {
     UNREACHABLE();
 }
 
+void Translator::DS_MAX(int bit_size, const GcnInst& inst) {
+    const IR::U32 addr{ir.GetVectorReg(IR::VectorReg(inst.src[0].code))};
+    const IR::U32 data{ir.GetVectorReg(IR::VectorReg(inst.src[1].code))};
+    const IR::U32 current_max{ir.GetVectorReg(IR::VectorReg(inst.dst[0].code))};
+    const IR::U32 max_value{ir.UMax(current_max, data)};
```

Comment:
Implementing shared atomics with normal UMin is not the same and will break things